src/components/SuggestionModal/styles.css

```diff
@@ -0,0 +1,41 @@
+.modal-container {
+  background-color: var(--color-background);
+  border-color: var(--color-white);
+  border-radius: 10px;
+  border-style: solid;
+  border-width: 1px;
+  color: var(--color-white);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  padding: 25px;
+  height: 50%;
+  width: 50%;
+}
+
+.field-container {
+  display: flex;
+  flex-direction: column;
+  padding: 10px 20px;
+}
+
+.field-label {
+  font-weight: 800;
+  padding-bottom: 10px;
+}
+
+.error-message {
+  color: var(--color-red);
+  font-weight: 600;
+  min-height: 22px;
+  padding-top: 10px;
+}
+
+.add-suggestion-button {
+  margin: 10px auto;
+  max-width: 12rem;
```

Comment:
use either px or rem, don´t mix them

Reply:
this one is the same change that is fixed from the last pr 👍🏻  I just moved it to a separate component, but I merge with the other change and fixed it here 😄